src/commands/Tags/tag.ts

```diff
@@ -41,87 +41,86 @@ export default class extends SkyraCommand {
 
 			const tagIndex = settings[GuildSettings.CustomCommands].findIndex((command) => command.id === id);
-			if (tagIndex === -1) throw language.get(LanguageKeys.Commands.Tags.TagNotExists, { tag: id });
+			if (tagIndex === -1) throw t(LanguageKeys.Commands.Tags.TagNotExists, { tag: id });
 
 			settings[GuildSettings.CustomCommands].splice(tagIndex, 1);
 
-			return language;
+			return t;
 		});
 
-		return message.send(language.get(LanguageKeys.Commands.Tags.TagRemoved, { name: id }));
+		return message.send(t(LanguageKeys.Commands.Tags.TagRemoved, { name: id }));
 	}
 
 	@requiresPermission(PermissionLevels.Moderator, async (message: GuildMessage) => {
-		throw await message.fetchLocale(LanguageKeys.Commands.Tags.TagPermissionLevel);
+		throw await message.resolveKey(LanguageKeys.Commands.Tags.TagPermissionLevel);
 	})
 	public async reset(message: GuildMessage) {
-		const language = await message.guild.writeSettings((settings) => {
+		await message.guild.writeSettings((settings) => {
 			settings[GuildSettings.CustomCommands].length = 0;
-			return settings.getLanguage();
 		});
 
-		return message.send(language.get(LanguageKeys.Commands.Tags.TagReset));
+		return message.sendTranslated(LanguageKeys.Commands.Tags.TagReset);
 	}
 
 	@requiresPermission(PermissionLevels.Moderator, async (message: GuildMessage) => {
-		throw await message.fetchLocale(LanguageKeys.Commands.Tags.TagPermissionLevel);
+		throw await message.resolveKey(LanguageKeys.Commands.Tags.TagPermissionLevel);
 	})
 	public async edit(message: GuildMessage, [id, content]: [string, string]) {
-		if (!content) throw await message.fetchLocale(LanguageKeys.Commands.Tags.TagContentRequired);
+		if (!content) throw await message.resolveKey(LanguageKeys.Commands.Tags.TagContentRequired);
 
-		const language = await message.guild.writeSettings((settings) => {
-			const language = settings.getLanguage();
+		const t = await message.guild.writeSettings((settings) => {
+			const t = settings.getLanguage();
 
 			const tagIndex = settings[GuildSettings.CustomCommands].findIndex((command) => command.id === id);
-			if (tagIndex === -1) throw language.get(LanguageKeys.Commands.Tags.TagNotExists, { tag: id });
+			if (tagIndex === -1) throw t(LanguageKeys.Commands.Tags.TagNotExists, { tag: id });
 
 			settings[GuildSettings.CustomCommands].splice(tagIndex, 1, this.createTag(message, id, content));
 
-			return language;
+			return t;
 		});
 
-		return message.send(language.get(LanguageKeys.Commands.Tags.TagEdited, { name: id, content: cutText(content, 1000) }));
+		return message.send(t(LanguageKeys.Commands.Tags.TagEdited, { name: id, content: cutText(content, 1000) }));
 	}
 
 	@requiredPermissions(['ADD_REACTIONS', 'EMBED_LINKS', 'MANAGE_MESSAGES', 'READ_MESSAGE_HISTORY'])
 	public async list(message: GuildMessage) {
 		// Get tags, prefix, and language
-		const [[tags, prefix], language] = await Promise.all([
+		const [[tags, prefix], t] = await Promise.all([
```

Comment:
Any reason this doesn't use guild.readSettings(settinga => [])?

Reply:
Who was the heretic person who did this?